src/utils/command-helpers.ts

```diff
@@ -59,30 +45,30 @@ export const USER_AGENT = `${name}/${version} ${platform}-${arch} node-${process
 /** A list of base command flags that needs to be sorted down on documentation and on help pages */
 const BASE_FLAGS = new Set(['--debug', '--http-proxy', '--http-proxy-certificate-filename'])
 
-export const NETLIFY_CYAN = chalk.rgb(40, 180, 170)
+// XXX(serhalp) Restore equivalent of `chalk.rgb(40, 180, 170)`
+export const NETLIFY_CYAN = picocolors.blueBright
```

Comment:
Just need to decide what to do about this...

Reply:
@serhalp the smaller drop-in replacement for `chalk` is `ansis` (~6 KB). Ansis has both the `ansis.hex('#28B4AA')` and `ansis.rgb(40, 180, 170)` functions.